doc/format-run.md

```diff
@@ -175,6 +175,22 @@ ordinary index type.
 The type of index in use is metadata that must be known for the whole LSM table
 handle. It does not vary per run.
 
+Independently of the concrete index serialization format, the type and the
+version of the format are stored at the beginning as a 32-bit number called the
```

Comment:
What do you mean by the first part of the sentence before the comma? The index serialization format is not stored anywhere

Reply:
I mean that, no matter what particular index serialization format is used, there is this 32-bit number at the beginning or, to put it differently, that all index serialization formats agree in having serialized indexes start with this 32-bit number. Does this need clarification?